ci_framework/roles/edpm_build_images/tasks/post.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.  
```

Comment:
Extra space here, will remove it in the next patch set.

afaik... it is generally sufficient to include the license in the main role files, specifically tasks/main.yml.

Including the license information in every task, file may lead to unnecessary duplication and could make it harder to manage and update the license information if it ever needs to be changed.

Reply:
remove license from here as its also missing in other task files and we have it in main.yaml